packages/pds/src/context.ts

```diff
@@ -46,9 +42,8 @@ export class AppContext {
       labelCache: LabelCache
       runtimeFlags: RuntimeFlags
       backgroundQueue: BackgroundQueue
-      appviewAgent?: AtpAgent
+      appviewAgent: AtpAgent
```

Comment:
Not suggesting a change in this PR at all, but in the future I'd be +1 on allowing the PDS to be configured without an appview url, in which case the `app.bsky` endpoints just don't exist on the PDS.

Reply:
totally agree 👍